Description: Avoid test-failing warnings on Python 3.12

Author: Rafael Santos, Hugo van Kemenade
Origin: upstream
Bug-Debian: https://bugs.debian.org/1056432
Forwarded: not-needed

--- a/openpyxl/packaging/core.py
+++ b/openpyxl/packaging/core.py
@@ -96,7 +96,7 @@ class DocumentProperties(Serialisable):
                  subject=None,
                  title=None,
                  ):
-        now = datetime.datetime.utcnow()
+        now = datetime.datetime.now(tz=datetime.timezone.utc)
         self.contentStatus = contentStatus
         self.lastPrinted = lastPrinted
         self.revision = revision
--- a/openpyxl/writer/excel.py
+++ b/openpyxl/writer/excel.py
@@ -289,7 +289,7 @@ def save_workbook(workbook, filename):
 
     """
     archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
-    workbook.properties.modified = datetime.datetime.utcnow()
+    workbook.properties.modified = datetime.datetime.now(tz=datetime.timezone.utc)
     writer = ExcelWriter(workbook, archive)
     writer.save()
     return True
--- a/openpyxl/writer/tests/test_excel.py
+++ b/openpyxl/writer/tests/test_excel.py
@@ -188,7 +188,7 @@ def test_modified(tmpdir):
     tmpdir.chdir()
 
     wb = Workbook()
-    modified = datetime.datetime(2011, 5, 19, 10, 23, 15)
+    modified = datetime.datetime(2011, 5, 19, 10, 23, 15,tzinfo=datetime.timezone.utc)
     wb.properties.modified = modified
 
     dest_filename = 'empty_book.xlsx'
